packages/nimble/R/cppDefs_cppProject.R

```diff
@@ -232,15 +229,16 @@ cppProjectClass <- setRefClass('cppProjectClass',
                                    },                                  
                                    compileFile = function(names, showCompilerOutput = nimbleOptions('showCompilerOutput'),
                                                           .useLib = UseLibraryMakevars) {
-                                       cppPermList <- c('RcppUtils.cpp',
-                                                        'Utils.cpp',
-                                                        'NamedObjects.cpp',
-                                                        'ModelClassUtils.cpp',
-                                                        'accessorClasses.cpp',
-                                                        'predefinedNimbleLists.cpp',
-                                                        'nimOptim.cpp'
-                                                        )
-                                       if(getNimbleOption('includeCPPdists')) cppPermList <- c(cppPermList, 'dists.cpp', 'nimDists.cpp')
+                                       ## cppPermList appears to be deprecated.  Will remove this commented-out block if no problems arise.
```

Comment:
I left it in this one PR while I wanted to be sure it passed testing. Was planning to remove it!